pkg/cmd/server/origin/master.go

```diff
@@ -390,10 +393,24 @@ func (c *MasterConfig) InitializeObjects() {
 	c.ensureOpenShiftSharedResourcesNamespace()
 }
 
-func (c *MasterConfig) InstallProtectedAPI(apiContainer *genericmux.APIContainer) ([]string, error) {
+func (c *MasterConfig) InstallProtectedAPI(apiserver *genericapiserver.GenericAPIServer) ([]string, error) {
+	apiContainer := apiserver.HandlerContainer
+
 	// initialize OpenShift API
 	storage := c.GetRestStorage()
 
+	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(buildapi.GroupName)
+	//if apiResourceConfigSource.AnyResourcesForVersionEnabled(rbacapiv1alpha1.SchemeGroupVersion) {
+	apiGroupInfo.VersionedResourcesStorageMap[buildapiv1.SchemeGroupVersion.Version] = map[string]rest.Storage{
```

Comment:
yeah, we should follow the upstream pattern here. This was just the easiest way to get the PoC going without refactoring half of origin.